### ISSUE TITLE
soviet训练和gpt训练报错无法使用，一键三连正常，显卡4060

### ISSUE THREAD

**Issue by landones** [RVC-Boss/GPT-SoVITS#399]
我按照大佬的教程在一步步操作，但不知道为什么训练就是无法成功，报错图片如下
![1](https://github.com/RVC-Boss/GPT-SoVITS/assets/67811286/f1a0d319-0529-417d-b914-d93e3f878603)
![2](https://github.com/RVC-Boss/GPT-SoVITS/assets/67811286/b89d22c4-b275-4830-8b98-c4c6ecda94c4)
![3](https://github.com/RVC-Boss/GPT-SoVITS/assets/67811286/d46faf39-2e7b-4f7d-9aaa-8a247fd97045)

**Comment by suonian** [comment]
啥系统啥配置，报内存，而且 pytorch 也告警UserWarning: Detected call of 'lr_scheduler.step()' before 'optimizer.step()'. In PyTorch 1.1.0 and later, you should call them in the opposite order: 'optimizer.step()' before 'lr_scheduler.step()'. Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.

**Comment by shaoqing404** [comment]
爆内存了，把batch size调低

**Comment by RVC-Boss** [comment]
可能是爆内存了，康康你的内存
2楼的告警不重要。
